src/pasta/fields/fp.rs

```diff
@@ -810,3 +835,42 @@ fn test_inv_2() {
 fn test_delta() {
     assert_eq!(Fp::DELTA, Fp::from(5).pow(&[1u64 << Fp::S, 0, 0, 0]));
```

Comment:
One or both of these:
```suggestion
    assert_eq!(Fp::DELTA, GENERATOR.pow(&[1u64 << Fp::S, 0, 0, 0]));
    assert_eq!(Fp::DELTA, Fp::multiplicative_generator().pow(&[1u64 << Fp::S, 0, 0, 0]));
```